sympy/integrals/laplace.py

```diff
@@ -775,7 +782,8 @@ def _laplace_apply_simple_rules(f, t, s):
                 debug('      f:     %s'%(f,))
                 debug('      rule:  %s o---o %s'%(t_dom, s_dom))
                 debug('      match: %s'%(ma, ))
-                return (s_dom.xreplace(ma), plane.xreplace(ma), c)
+                return (s_dom.xreplace(ma).subs({s_: s}),
+                        plane.xreplace(ma), c.subs({s_: s}))
```

Comment:
```suggestion
                return (s_dom.xreplace(ma).subs({s_: s}),
                        plane.xreplace(ma).subs({s_: s}), S.true)
```
Does `c` return other than `True`?
I also think that `subs` should go to `plane` rather than `c` if it is a mistake.
It is interesting why the existing test cases didn't point out that though. But I believe conditions only apply for coefficients than the symbols, so we can probably ignore

Reply:
> Does `c` return other than `True`?

Of course not, this comes after `if c==True:`, I feel a bit stupid right now :))) 

> I also think that `subs` should go to `plane` rather than `c` if it is a mistake. 

No, `plane` is always a number and can never contain the transform variable, so no substitution is necessary.